src/components/card/growing.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { FC, ReactNode } from "react"
+import classNames from "classnames"
+
+interface Props {
+  renderImage: () => ReactNode
+  children: ReactNode
+  noMarginsOnMobile?: boolean
+}
+
+export const GrowingCard: FC<Props> = ({
+  renderImage,
+  children,
+  noMarginsOnMobile,
```

Comment:
It's not really related to margins, isn't it? More like `roundedCornersOnMobile`

Reply:
I did not catch that, but now that you point at it - the name here seems overly specific. can we generalize it more? what would be a "semantic" approach? we have for example a `fullWidth` property on the `AlertMessage` https://github.com/carforyou/carforyou-components-pkg/blob/136ab6c03c84012f37c24568b09a566f80904819/src/components/alertMessage.tsx#L38

if `fullWidth` -> non-rounded corners
else -> rounded corners

Reply:
oh my. we closed this PR already 🙈